.travis.yml

```diff
@@ -3,7 +3,7 @@ language: c
 sudo: required
 
 before_install:
-  - sudo apt-get install -y --no-install-recommends python-numpy-dev
+  - sudo apt-get install -y --no-install-recommends python-numpy-dev texlive texlive-fonts-extra # fonts-extra needed for compiling R package documentation (inconsolata font)
```

Comment:
R package is checking OK again now after this change -- it needs latex and some special font